Gemfile

```diff
@@ -46,18 +46,17 @@ if !defined?(JRUBY_VERSION) && RUBY_VERSION < '2.5'
   gem 'google-protobuf', '< 3.12'
 end
 gem 'grpc' if !defined?(JRUBY_VERSION) && RUBY_VERSION < '3.0'
-gem 'json'
+gem 'json', '2.5.1'
```

Comment:
Needs a note that 2.6.1 has a segfault with ruby >=3.0.4, can be unpinned after a new release fixes that issue